local-dev/templates/mod_security_ignore.conf

```diff
@@ -0,0 +1,3 @@
+SecRule REQUEST_URI "@contains importEntities" phase:1,id:'1000000001',nolog,allow,ctl:ruleEngine=Off,ctl:auditEngine=Off
+SecRule REQUEST_URI "@contains batchUpsert" phase:1,id:'1000000002',nolog,allow,ctl:ruleEngine=Off,ctl:auditEngine=Off
+SecRule REQUEST_URI "@contains loading..." phase:1,id:'1000000003',nolog,allow,ctl:ruleEngine=Off,ctl:auditEngine=Off
```

Comment:
where did these three entries come from?

Reply:
Thanks for taking look! I created this PR by working backwards - render configuration from firecloud-develop using the [firecloud-setup.sh](https://github.com/broadinstitute/firecloud-develop/blob/dev/run-context/local/scripts/firecloud-setup.sh) script, then reconstruct the output in the Rawls repo with secrets pulled from Vault. This template comes from [common templates](https://github.com/broadinstitute/firecloud-develop/blob/dev/base-configs/common/mod_security/mod_security_ignore.conf). I'm happy to remove it (and anything else in this PR) that seems unnecessary.